ui/style/style_core_font.cpp

```diff
@@ -168,7 +200,18 @@ void StartFonts() {
 		//QFont::insertSubstitution(name, fallback);
 #endif // Q_OS_WIN
 	}
-#endif // !DESKTOP_APP_USE_PACKAGED_FONTS
+
+#ifdef Q_OS_WIN
+	auto list = QStringList();
+	list.append("Microsoft YaHei");
```

Comment:
Why it's Microsoft Yahei instead of Microsoft Yahei UI?

Reply:
I took a list of fonts from FontMod readme